fun.py

```diff
@@ -195,9 +195,10 @@ def pizza(self):
             yield (pixels, 0.1, False)
 
     def fade(self):
-      bright = 255
-      pixels = [(bright,bright,bright)] * numLEDs
-      while bright > 0:
-        bright -= 1
-        pixels = [(bright,bright,bright)] * numLEDs
-        self.client.put_pixels(pixels)
+        whiile True:
+            bright = 255
+            pixels = [(bright,bright,bright)] * numLEDs
+            while bright > 0:
+                bright -= 1
```

Comment:
bright gets decremented all the way down to 0, but it's never reflected in the pixels, or pushed to the pixels.
Consider indenting the bottom two lines into the while loop
```
         ->  pixels = [(bright,bright,bright)] * numLEDs
         ->   yield (pixels, 3, False)
```